src/elements/content-sidebar/activity-feed/activity-feed/ActivityFeed.scss

```diff
@@ -73,8 +73,10 @@
         }
 
         .bcs-activity-feed-annotation-activity.bcs-is-focused {
+            margin-bottom: 1px;
             padding-left: 21px;
             border-left: 4px solid $bdl-yellorange;
+            animation: none;
```

Comment:
Rather than overriding, can we avoid applying the animation in the first place?

Reply:
I don't think we should remove the animation at this point. I could create a new rule for the annotation activity cards but that seems a bit overkill just for the animation. Maybe we should just keep the animation and update the animation behavior at a later time.